src/test/java/org/symphonyoss/symphony/messageml/elements/form/DatePickerTest.java

```diff
@@ -245,6 +245,30 @@ public void sendFieldTooLongOnMessageML() throws Exception {
     context.parseMessageML(input,null, MessageML.MESSAGEML_VERSION);
   }
 
+  @Test
+  public void sendUnsupportedDateFormat() throws Exception {
```

Comment:
might be worth testing the different separator you handle (like . /)
it might fail as we use DateTimeFormatter.ofPattern (not entirely sure)

Reply:
Done